storage/jpa/src/main/java/io/apicurio/registry/storage/impl/jpa/entity/Rule.java

```diff
@@ -56,5 +56,5 @@
     private String artifactId;
 
     @Column(name = "name", updatable = false, nullable = false)
-    private RuleType name;
+    private String name;
```

Comment:
Why plain string and not enum?

Reply:
Hibernate had some problems with the enum, causing errors. I can probably look into in and use the enum if you think it's better that way.